backend/settings/urls.py

```diff
@@ -11,10 +11,6 @@
 from rest_framework import routers
 
 from apps.github.api.urls import router as github_router
-from apps.owasp.api.search.chapter import chapters as search_chapters
-from apps.owasp.api.search.committee import committees as search_committees
-from apps.owasp.api.search.issue import project_issues as search_project_issues
-from apps.owasp.api.search.project import projects as search_projects
```

Comment:
Please take care of the handlers (e.g. [this](https://github.com/OWASP/Nest/blob/f116c277f7db10ae0efd7c97132f40b9a620c013/backend/apps/owasp/api/search/chapter.py#L38) and [that](https://github.com/OWASP/Nest/blob/f116c277f7db10ae0efd7c97132f40b9a620c013/backend/apps/owasp/api/search/committee.py#L33)) and related tests too.